src/api/app/controllers/source_controller.rb

```diff
@@ -1282,7 +1282,9 @@ def package_command_collectbuildenv
   def package_command_instantiate
     project = Project.get_by_name(params[:project])
     opackage = Package.get_by_project_and_name(project.name, params[:package], {check_update_project: true})
-
+    unless opackage
+      raise RemoteProjectError.new "Instantiation from remote project is not supported"
```

Comment:
opackage is only nil when it is a remote package, otherwise you get an exception.